bigframes/core/compile/googlesql/expression.py

```diff
@@ -84,15 +113,41 @@ class AliasExpression(Expression):
     alias: str
 
     def sql(self) -> str:
-        return f"`{_escape_chars(self.alias)}`"
+        return identifier(self.alias)
 
 
 @dataclasses.dataclass
 class CTEExpression(Expression):
     name: str
 
     def sql(self) -> str:
-        return f"`{_escape_chars(self.name)}`"
+        return identifier(self.name)
+
+
+@dataclasses.dataclass
+class Cast(Expression):
+    value: ScalarExpression | str
+    to_type: str
+
+    def sql(self):
+        value_sql = self.value if isinstance(self.value, str) else self.value.sql()
+        return f"CAST({value_sql} AS {self.to_type})"
+
+
+@dataclasses.dataclass
+class Timestamp(Expression):
```

Comment:
Move this function to `functions/timestamp.py` folder. Also add docstring `This class represents GoogleSQL `TIMESTAMP` syntax.`